docs/reference/data-streams/tsds.asciidoc

```diff
@@ -0,0 +1,298 @@
+* <<number,`integer`>>
+* <<number,`long`>>
+* <<number,`unsigned_long`>>
+
+[[dimension-limits]]
+.Dimension limits
+****
+In a TSDS, {es} uses dimensions to generate the document `_id` and
+<<tsid,`_tsid`>> values. To prevent overly large values, {es} limits the number
+of dimensions for an index using the
+<<index-mapping-dimension-fields-limit,`index.mapping.dimension_fields.limit`>>
+index setting. While you can increase this limit, the resulting document `_id`
+and `_tsid` values can't exceed 32KB.
+****
+
+[discrete]
+[[time-series-metric]]
+==== Metrics
+
+Metrics are fields that contain numeric measurements. While not required,
+documents in a TSDS typically contain one or more metric fields.
+
+A field can't be both a dimension and a metric. Unlike dimensions, the metric
+values for a time series typically change over time, even if rarely or slowly. 
+
+To mark a field as a metric, you must specify a metric type using the
+`time_series_metric` mapping parameter. The following field types support the
+`time_series_metric` parameter:
+
+* <<aggregate-metric-double,`aggregate_metric_double`>>
+* <<histogram,`histogram`>>
+* All <<number,numeric field types>>
+
+Accepted metric types vary based on the field type:
+
+.Valid values for `time_series_metric`
+[%collapsible%open]
+====
+// tag::time-series-metric-counter[]
+`counter`:: A number that only increases or resets to `0` (zero). For
+example, a count of errors or completed tasks.
+// end::time-series-metric-counter[]
++
+Only numeric and `aggregate_metric_double` fields support the `counter` metric
+type.
+
+// tag::time-series-metric-gauge[]
+`gauge`:: A number that can increase or decrease. For example, a temperature or
+available disk space.
+// end::time-series-metric-gauge[]
++
+Only numeric and `aggregate_metric_double` fields support the `gauge` metric
+type.
+
+// tag::time-series-metric-histogram[]
+`histogram`:: A pair of numeric arrays that measure the distribution of values
+across predefined buckets. For example, server response times by percentile.
+// end::time-series-metric-histogram[]
++
+Only `histogram` fields support the `histogram` metric type.
+
+// tag::time-series-metric-summary[]
+`summary`:: An array of aggregated values, such as `sum`, `avg`, `value_count`,
+`min`, and `max`.
+// end::time-series-metric-summary[]
++
+Only `aggregate_metric_double` fields support the `gauge` metric type.
+
+// tag::time-series-metric-null[]
+`null` (Default):: Not a time series metric.
+// end::time-series-metric-null[]
+====
+
+[discrete]
+[[time-series-mode]]
+=== Time series mode
+
+The matching index template for a TSDS must contain a `data_stream` object with
+the `index_mode: time_series` option. This option ensures the TSDS creates
+backing indices with an <<index-mode,`index.mode`>> setting of `time_series`.
+This setting enables most TSDS-related functionality in the backing indices.
+
+If you convert an existing data stream to a TSDS, only backing indices created
```

Comment:
Thanks. I'd really appreciate if you can double check. James said mentioned in a handoff document that `Rather than publicly call it TSDB (which can be confusing), we’ve decided to call the feature time series data stream (TSDS).` but I'm not sure who was part of that conversation.